apps/api/src/app/events/e2e/echo-trigger.e2e-ee.ts

```diff
@@ -389,6 +390,55 @@ describe('Echo Trigger ', async () => {
     expect(messagesAfter.length).to.be.eq(1);
     expect(messagesAfter[0].content).to.match(/people waited for \d+ seconds/);
   });
+
+  it('should trigger the scheduled delay with invalid date', async () => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ Maybe a better name would be 'should report error on date that is in the past from now'